publisher-sdk/src/main/java/com/criteo/publisher/csm/Metric.java

```diff
@@ -68,6 +68,14 @@
   @Nullable
   abstract String getRequestGroupId();
 
+  /**
+   * Zone ID that was mapped during bidding to the CDB slot response.
+   * <p>
+   * This should be present in case of valid bid.
```

Comment:
should or must? https://www.ietf.org/rfc/rfc2119.txt

Reply:
Normally it is "must". But we do not control the TLA and this information comes from it.